examples/options/include/traccc/options/program_options.hpp

```diff
@@ -28,7 +29,7 @@ class program_options {
     program_options(
         std::string_view description,
         const std::vector<std::reference_wrapper<interface> >& options,
-        int argc, char* argv[]);
+        int argc, char* argv[], std::unique_ptr<const traccc::Logger> ilogger);
```

Comment:
Why no default argument here? I think a default name could be conveniently given for messages coming from this code.

Reply:
See message below, but I will add a default argument.

Reply:
Default added.